highlight/highlight/langDefs/chpl.lang

```diff
@@ -11,16 +11,16 @@ Description="Chapel"
 
 Keywords={
   { Id=1,
-   List={"as", "align", "atomic", "begin", "break", "by", "class", "cobegin",
-         "coforall", "config", "const", "continue", "proc", "iter", "delete",
-         "dmapped", "do", "domain", "else", "enum", "except", "export",
-         "extern", "false", "for", "forall", "if", "in", "index", "inline",
-         "inout", "label", "lambda", "let", "local", "module", "new", "nil",
-         "noinit", "on", "only", "otherwise", "out", "param", "private",
+   List={"as", "align", "atomic", "begin", "break", "by", "catch", "class",
+         "cobegin", "coforall", "config", "const", "continue", "proc", "iter",
```

Comment:
It bothers me that proc and iter are not alphabetically sorted, but that isn't your fault (and I am sad I didn't notice it before now)